docs/areatrigger_tavern.md

```diff
@@ -11,7 +11,7 @@ Enable a trigger when player enters a city or tavern. This causes the player to
 | Field     | Type         | Attributes | Key | Null | Default | Extra | Comment    |
 |-----------|--------------|------------|-----|------|---------|-------|------------|
 | [id][1]   | mediumint(8) | unsigned   | PRI | NO   | 0       |       | Identifier |
-| [name][2] | text         | signed     |     | YES  | NULL    |       |            |
+| [name][2] | text         |            |     | YES  |         |       |            |
```

Comment:
I suggest to leave NULL

Reply:
I think we are facing the same case.
And those errors, are mine, because when I converted from HTML to Markdown, in many occasions, it appeared (NULL) and I thought it meant that they were null. But now that I am reviewing the scripts, I realize that they were not like that.

```sql
CREATE TABLE `areatrigger_tavern` (
  `id` mediumint(8) unsigned NOT NULL DEFAULT '0' COMMENT 'Identifier',
  `name` text,
  PRIMARY KEY (`id`)
) ENGINE=MyISAM DEFAULT CHARSET=utf8 ROW_FORMAT=FIXED COMMENT='Trigger System'
```
Thanks for the feedback.
If I am wrong, please let me know, to correct the error.